test/jdk/java/net/httpclient/BasicAuthTest.java

```diff
@@ -95,16 +105,18 @@ public static void main(String[] args) throws Exception {
 
             // try a POST
 
-            req = HttpRequest.newBuilder(uri)
-                             .POST(BodyPublishers.ofString(POST_BODY))
+            req = builder.copy().POST(BodyPublishers.ofString(POST_BODY))
                              .build();
+            System.out.println("\n\nSending POST request: " + req);
             resp = client.send(req, BodyHandlers.ofString());
             ok = resp.statusCode() == 200;
 
             if (!ok || ca.count != 1)
                 throw new RuntimeException("Test failed");
```

Comment:
Hello Daniel, while we are changing this test, do you think we should also update this exception message, here and other places where we throw this, to also include the `ca.count` in the exception message to help debug any potential failures?

Reply:
Done